assets/stories/skate-design/Typography.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Unstyled } from "@storybook/blocks"
+import "../../css/storybook_reboot.scss"
+
+<Meta title="Typography" />
+
+# Global Settings / Style
+
+Examples of typography styles found in Skate, based on defaults from
+Reboot with some modifications.
+
+<Unstyled>
+  <p>
+    {
+      "The default font is Inter. Body text is 1rem, 400 weight, $gray-900(#141524)."
+    }
+    <br />
+    {"In paragraphs, we use 150% line height."}
+  </p>
+  <p>Between paragraphs, there is 1rem of spacing.</p>
+  <p>
+    <span>
+      <a href="#">Link text</a> is also 1rem, 400 weight, using the primary
+      color ($eggplant-500).
+    </span>
+  </p>
+</Unstyled>
+
+# Headings
+
+<Unstyled>
+  <h1>h1 Heading</h1>
+  <h2>h2 Heading</h2>
+  <h3>h3 Heading</h3>
+  <h4>h4 Heading</h4>
+  <h5>h5 Heading</h5>
+  <h6>h6 Heading</h6>
+</Unstyled>
+
+# Lead Paragraph
+
+<Unstyled>
+  <p class="lead">
+    <span>
+      This is a lead paragraph. Lead paragraphs stand out from regular
+      paragraphs with 300 weight, 1.25rem font size, and 1.5 line height.
+    </span>
+  </p>
+</Unstyled>
+
+# Inline Text Elements
+
+<Unstyled>
+You can use the <mark>mark tag</mark> to highlight text.
+
+You can use the <del>deleted tag</del> for deleted text.
+
+You can use the <s>strikethrough tag</s> for text that is no longer relevant or accurate.
+
+You can use the <ins>inserted tag</ins> for text that has been added to the document.
+
+You can use the <u>underline tag</u> for underlined text.
+
+You can use the <small>small tag</small> for text that is to be treated as fine print.
+
+You can use the <strong>strong tag</strong> for bold text.
+
+You can use the <em>emphasis tag</em> for italicized text.
```

Comment:
suggestion: I like this! What do you think about adding a `<Source/>` block which shows this code so it's more explicit where the tags are in the text, and how it's being used?

Reply:
It looks like the `<Source/>` block is actually a little idiosyncratic (you can't just wrap it around code and get what you'd expect, to insert literal source code you need to pass it in as a prop): https://storybook.js.org/docs/api/doc-block-source

How do you feel about just putting it in a `<code>` block instead?

Reply:
I'd be interested to see a solution using `<code>`, but I'm not sure you're going to get what you expect given the markdown parser is also present.

A few things I like about the `<Source/>` element is
1. It can skip out on the markdown rendering by passing in the code as a prop string
1. it comes with some decent styling 
![image](https://github.com/mbta/skate/assets/6913198/5489406f-7bb2-45ef-a2ff-026736733489)
1. It has syntax highlighting

It's not quite perfect, and if you'd prefer something else I'd be interested to see.

Unfortunately, you can't just take the existing markdown and put it in the source block, because it won't have the `<p>` tags, but it's also fine without them IMO.

```mdx
<Source language="html" code={`
You can use the <mark>mark tag</mark> to highlight text.

You can use the <del>deleted tag</del> for deleted text.

You can use the <s>strikethrough tag</s> for text that is no longer relevant or accurate.

You can use the <ins>inserted tag</ins> for text that has been added to the document.

You can use the <u>underline tag</u> for underlined text.

You can use the <small>small tag</small> for text that is to be treated as fine print.

You can use the <strong>strong tag</strong> for bold text.

You can use the <em>emphasis tag</em> for italicized text.
`}/>
```